Resources/Server Info/RP_Rules.txt

```diff
@@ -1,103 +1,119 @@
+    - You determine that your life or the life of an innocent is in immediate danger
+    - The suspect is unable to be safely detained by less-lethal means
+    - If no other reasonable options are readily available and allowing the suspect to continue would be an unreasonable danger to the station/crew
+
+Security/Command will be expected to answer for use of lethal force. Security/Command will be expected to effect arrests on criminals and prevent them from dying while in custody, even if lethal force is used, unless there is strong reason to believe the criminal is an antagonist. Security/Command is strongly encouraged, but not required, to clone antagonists and effect a permabrigging or other sentence as deemed appropriate.
+
+[color=#a4885c]20.[/color] Security/Command are expected to protect detainees in their custody to the best of their ability so as long as it does not come to unreasonable risk to themselves, the crew or the station at large to do so.
+    - Brig times should generally not exceed 10 minutes. Repeat offenders or antagonists may be permabrigged.
+    - Detainees that die in your custody must be cloned unless they have been (legally) executed or there is strong reason to believe they are an antagonist.
+    - Executions must be for a permabriggable crime and approved by the Captain/Acting Captain, who will answer for approving it alongside Security's chain of command. Security is beholden to [color=#FFFFFF]Space Law.[/color].
+
+[color=#ff0000]RP SERVER SPECIFIC RULES[/color]
+
+[color=#a4885c]21.[/color] This server is a "Roleplay Expected" server. We define this as performing your assigned duties, "doing your job". This means it is important to do what is expected out of your department and not what would be expected out of other departments.
     - Don’t do other peoples jobs for them. Opt in for the role you intend to play or change your job by visiting the Head of Personnel. Failure to do your basic duties may result in a job ban.
 
-[color=#a4885c]15.[/color] Each round exists independently of all other rounds. While friendships persisting through rounds is okay, using experiences from other rounds to give an advantage or disadvantage to other players is not permitted.
+[color=#a4885c]22.[/color] Each round exists independently of all other rounds. While friendships persisting through rounds is okay, using experiences from other rounds to give an advantage or disadvantage to other players is not permitted.
 
-[color=#a4885c]16.[/color] New Life Rule is in effect - Do not use information obtained while dead or from a previous life upon returning to the land of the living. Even if you are cloned, you do not remember the events culminating in your death.
+[color=#a4885c]23.[/color] New Life Rule is in effect - Do not use information obtained while dead or from a previous life upon returning to the land of the living. Even if you are cloned, you do not remember the events culminating in your death.
```

Comment:
This is worded confusingly. I first read this as a contradiction to line 45.

```suggestion
[color=#a4885c]23.[/color] New Life Rule is in effect - Do not use information obtained while dead or from a previous life upon returning to the land of the living. Even if you are cloned, you do not remember the events culminating during your death; you may only recall events up to the point you lost consciousness.
```

Reply:
Probably better to just delete this since this is already covered by the "no metagaming" rule.